Minio/DataModel/ObjectOperationsResponse.cs

```diff
@@ -215,26 +215,28 @@ internal NewMultipartUploadResponse(HttpStatusCode statusCode, string responseCo
     internal string UploadId { get; }
 }
 
-internal class PutObjectResponse : GenericResponse
+public class PutObjectResponse : GenericResponse
 {
-    internal string Etag;
+    public string Etag;
+    public string ObjectName;
+    public string Size;
 
-    internal PutObjectResponse(HttpStatusCode statusCode, string responseContent,
-        Dictionary<string, string> responseHeaders)
+    public PutObjectResponse(HttpStatusCode statusCode, string responseContent,
+        Dictionary<string, string> responseHeaders, string size, string name)
```

Comment:
It would be useful to have the response headers exposed for end user interrogation.

Reply:
Done